external/buildscripts/build_runtime_stv.sh

```diff
@@ -20,7 +20,7 @@ for STV_TARGET in ${STV_TARGETS}; do
 	STV_GCC_PREFIX="${!STV_GCC_PREFIX}"
 
 	# need to swap flags when building for different platforms.
-	CXXFLAGS="-g -DARM_FPU_VFP=1 -DHAVE_ARMV6=1 -D__ARM_EABI__ -DLINUX -D__linux__ -DHAVE_PTHREAD_MUTEX_TIMEDLOCK -march=armv7-a -mfpu=vfp -mfloat-abi=softfp -fpic -funwind-tables -ffunction-sections -fdata-sections";
+	CXXFLAGS="-g -DARM_FPU_VFP=1 -DHAVE_ARMV6=1 -D__ARM_EABI__ -DLINUX -D__linux__ -DPLATFORM_STV -DHAVE_PTHREAD_MUTEX_TIMEDLOCK -march=armv7-a -mfpu=vfp -mfloat-abi=softfp -fpic -funwind-tables -ffunction-sections -fdata-sections";
```

Comment:
I think you could also just define USE_PTHREAD_LOCKS and get the same result without changing the header file. Anyone prefer one or the other?